app/ui/legacy/src/main/java/com/fsck/k9/ui/K9Drawer.kt

```diff
@@ -83,6 +84,9 @@ class K9Drawer(private val parent: MessageList, savedInstanceState: Bundle?) : K
     private var openedAccountUuid: String? = null
     private var openedFolderId: Long? = null
 
+    // TODO: unifiedInbox needed to get its message counts from back end
+    // private val unifiedInbox: SearchAccount = SearchAccount.createUnifiedInboxAccount()
```

Comment:
Please don't leave any commented out code in the pull request. If a feature isn't ready, it doesn't go into the code base.

Reply:
Ok. Done.